DllImportGenerator/DllImportGenerator/DllImportStub.cs

```diff
@@ -217,7 +218,7 @@ public class GeneratedDllImportData
             var additionalAttrs = new List<AttributeListSyntax>();
 
             // Define additional attributes for the stub definition.
-            if (env.TargetFrameworkVersion >= new Version(5, 0))
+            if (env.TargetFrameworkVersion >= new Version(5, 0) && !IsSkipLocalsInit(method))
```

Comment:
Do we still need the target framework version check?

Reply:
We still have our tests around validating against downlevel usage, so I'd like to keep the target framework version check until we drop that.